modular_zubbers/code/modules/mob/check_pulse.dm

```diff
@@ -0,0 +1,27 @@
+/mob/verb/check_pulse(atom/A as mob in view())
```

Comment:
We no longer use single letter vars, and ghosts are very obviously dead so even if a wizard made them visible there's no reason to check them.
```suggestion
/mob/verb/check_pulse(atom/checked_atom as mob/living in view())
```